src/lucky_flow.cr

```diff
@@ -147,16 +148,6 @@ class LuckyFlow
   end
 
   def self.default_driver_path
-    Process.find_executable("chromedriver") || "#{__DIR__}/../vendor/chromedriver-80-#{os}"
-  end
-
-  private def self.os
-    {% if flag?(:linux) %}
-      "linux64"
-    {% elsif flag?(:darwin) %}
-      "mac64"
-    {% else %}
-      raise "This OS is not supported yet."
-    {% end %}
+    Webdrivers::Chromedriver.install
```

Comment:
Does this raise an error if it can't install? If so I think it'd be awesome to rescue it and raise a custom error message that outputs the original error message *and* how to set the webdriver path with Habitat setting.

I think normally it should work, but this might be helpful just in case something does break

```crystal
Something went wrong while installing Chromedriver:

  #{og_error_message}

If you'd like to manually install Chromedriver yourself, make sure to tell LuckyFlow where it is located:

  LuckyFlow.configure do |settings|
    settings.chromedriver_path = "/path/to/chromedriver"
  end
```